rasterio/_drivers.pyx

```diff
@@ -7,31 +7,9 @@ import sys
 
 from rasterio.compat import string_types
 
+from rasterio cimport _gdal
```

Comment:
@sgillies could you use `from rasterio._gdal cimport GDALGetDriverCount, ...` instead?